elliott/elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -56,9 +62,98 @@ async def attach_cve_flaws_cli(
     1851422, 1866148, 1858981, 1852331, 1861044, 1857081, 1857977, 1848647,
     1849044, 1856529, 1843575, 1840253]
     """
+
     if sum(map(bool, [advisory_id, default_advisory_type, into_default_advisories])) != 1:
         raise click.BadParameter("Use one of --use-default-advisory or --advisory or --into-default-advisories")
     runtime.initialize()
+
+    if runtime.build_system == 'konflux':
+        if into_default_advisories or advisory_id:
+            raise click.UsageError(
+                "Konflux does not yet support --into-default-advisories or --advisory options, "
+                "please use --use-default-advisory instead"
+            )
+
+        release_notes = await handle_konflux_cve_flaws(runtime, default_advisory_type)
+        click.echo(json.dumps(release_notes.model_dump(mode='json'), indent=4))
```

Comment:
An option or default to yaml dump would be good, it will be consistent with shipment yaml and will make it easier for us to copy paste when doing things by hand

Reply:
Done with 1fb21b3407fde716bfbc33d04679b31561493c71